packer/scripts/create-k8s-certs.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
(For the file name) Vlad doesn't like `k8s` and prefers `kube` instead. Don't get it, but meh, might as well.